src/assets/i18n/en.json

```diff
@@ -318,6 +306,10 @@
 	"Section": "Section name",
 	"Detail label": "Field label",
 	"Detail field": "Used field",
+	"Quicklook url description": "",
+	"Quicklook description description": "",
```

Comment:
is it normal to have duplicated 'description'

Reply:
Yes it is for the label of the form that is not used in the current table format of the quicklook form. You have a description for each of the fields that are in the quicklook form.